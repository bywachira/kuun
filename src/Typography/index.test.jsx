import React from 'react'
import enzyme, { mount } from 'enzyme'
import toJSON from 'enzyme-to-json'
import adapter from 'enzyme-adapter-react-16'
import { Title, Paragraph } from '.'

enzyme.configure({ adapter: new adapter() })

describe('#TYPOGRAPHY_TITLE', () => {
  it('should render', () => {
    const component1 = mount(<Title size={4}>Tiny Title</Title>)
    const component2 = mount(<Title size={3}>Small Title</Title>)
    const component3 = mount(<Title size={2}>Medium Title</Title>)
    const component4 = mount(<Title>Large Title</Title>)

    expect(toJSON(component1)).toMatchSnapshot()
    expect(toJSON(component2)).toMatchSnapshot()
    expect(toJSON(component3)).toMatchSnapshot()
    expect(toJSON(component4)).toMatchSnapshot()
  })
})

describe('#TYPOGRAPHY_PARAGRAPH', () => {
  it('should render', () => {
    const component = mount(<Paragraph strong={true} color='red' />)

    expect(toJSON(component)).toMatchSnapshot()
  })
})
