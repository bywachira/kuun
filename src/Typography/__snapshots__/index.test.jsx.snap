// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#TYPOGRAPHY_PARAGRAPH should render 1`] = `
<Paragraph
  color="red"
  strong={true}
>
  <styled.p
    color="red"
    strong={true}
  >
    <p
      className="sc-eCApGN bZjrIc"
      color="red"
    />
  </styled.p>
</Paragraph>
`;

exports[`#TYPOGRAPHY_TITLE should render 1`] = `
<Title
  size={4}
>
  <styled.h4>
    <h4
      className="sc-hKFyIo fZBqiv"
    >
      Tiny Title
    </h4>
  </styled.h4>
</Title>
`;

exports[`#TYPOGRAPHY_TITLE should render 2`] = `
<Title
  size={3}
>
  <styled.h3>
    <h3
      className="sc-dlnjPT IkCFx"
    >
      Small Title
    </h3>
  </styled.h3>
</Title>
`;

exports[`#TYPOGRAPHY_TITLE should render 3`] = `
<Title
  size={2}
>
  <styled.h2>
    <h2
      className="sc-gtssRu jscVVC"
    >
      Medium Title
    </h2>
  </styled.h2>
</Title>
`;

exports[`#TYPOGRAPHY_TITLE should render 4`] = `
<Title>
  <styled.h1>
    <h1
      className="sc-bdnylx jClOke"
    >
      Large Title
    </h1>
  </styled.h1>
</Title>
`;
